intermediate/micro-frontend.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Scalable Frontend Architecture
```

Comment:
Can you change the title to "MicroFrontend: Scalable Frontend Architecture". It's a known and intriguing topic - will help get more viewers as people search for it

And also add it as a tag microfrontend, micro-frontend